filament/src/details/Scene.h

```diff
@@ -102,7 +102,8 @@ class FScene : public Scene {
 
     enum {
         RENDERABLE_INSTANCE,    //  4 | instance of the Renderable component
-        WORLD_TRANSFORM,        // 16 | instance of the Transform component
+        WORLD_TRANSFORM,        // 64 | instance of the Transform component
```

Comment:
Nit: let's revert to their 16 so that when they decide what they want to signify here (bytes VS floats VS DWORDs) and fix it, we won't conflict with them.

Reply:
Fixed in a1c7b04